core/src/main/java/org/mule/runtime/core/privileged/processor/chain/AbstractMessageProcessorChain.java

```diff
@@ -185,25 +185,34 @@ public CoreEvent process(CoreEvent event) throws MuleException {
   private BiConsumer<Throwable, Object> getContinueStrategyErrorHandler(Processor processor) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor this method to reduce its Cognitive Complexity from 21 to the 15 allowed. ([squid:S3776](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS3776))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7683&issues=AWjyMA3aotIXnmsUSXGi&open=AWjyMA3aotIXnmsUSXGi)

[ISSUE_KEY]: AWjyMA3aotIXnmsUSXGi